### PR TITLE
Feature/add tools

### PR DESCRIPTION
**Pull request by mspector**

### CODE REVIEW

File: openreview/tools.py [hunk 1]
```diff
@@ -51,6 +53,99 @@ def create_profile(client, email, first, last, middle = None):
         raise openreview.OpenReviewException('There is already a profile with this email: {0}'.format(email))
 
 
+def get_or_create_profile(client, email, first, last, allow_duplicates=False, verbose=False):
```
Comment (melisabok) [inline openreview/tools.py]: I think create profile is doing this, why we need another funciton?

File: openreview/tools.py [hunk 2]
```diff
@@ -194,3 +289,44 @@ def get_paper_conflicts(client, paper):
...
+            ids.append(member)
+
+    group.members = ids + emails
+    client.post_group(group)
+
+def get_all_notes(client, invitation, limit=1000):
+    done = False
+    notes = []
+    while not done:
+        batch = client.get_notes(invitation=invitation, limit=limit)
```
Comment (melisabok) [inline openreview/tools.py]: you need to pass an offset otherwise you get always the same result.

File: openreview/tools.py (hunk 1 above)
Comment (mspector) [reply openreview/tools.py]: ah yes, you're right. I'm not sure why I missed that

Comment (melisabok) [reply openreview/tools.py]: don't worry it is not urgent to fix this.

File: openreview/tools.py (hunk 2 above)
Comment (mspector) [reply openreview/tools.py]: huh, you are right, that's weird. the test I did must have been flawed somehow.

**Pull request merged**